sdk/lib/opentelemetry/sdk/trace/export/in_memory_span_exporter.rb

```diff
@@ -66,6 +66,7 @@ def export(spans)
             @mutex.synchronize do
               return FAILED_NOT_RETRYABLE if @stopped
 
+              spans = spans.entries unless spans.is_a? Array
```

Comment:
I had to look this up. I feel like an unconditional `@finished_spans.concat(spans.to_a)` below is clearer. It'll do the right thing for both Array and non-Array Enumerables.

Reply:
Updated